From: Ian Ferguson <iferguso@genome.wustl.edu>
Date: Mon, 2 Mar 2015 21:20:14 +0100
Subject: [PATCH] No git in Makefile

Don't try to figure out the current git revision as part of the build
process.
---
 Makefile |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 82ed69c..7add072 100644
--- a/Makefile
+++ b/Makefile
@@ -63,10 +63,6 @@ all:$(PROG) plugins
 
 # See htslib/Makefile
 PACKAGE_VERSION = 1.1
-ifneq "$(wildcard .git)" ""
-PACKAGE_VERSION := $(shell git describe --always --dirty)
-version.h: $(if $(wildcard version.h),$(if $(findstring "$(PACKAGE_VERSION)",$(shell cat version.h)),,force))
-endif
 version.h:
 	echo '#define BCFTOOLS_VERSION "$(PACKAGE_VERSION)"' > $@
 
-- 
